src/full-screen/components/runtime.tsx

```diff
@@ -25,6 +25,14 @@ export const Runtime: React.FC = () => {
       window.removeEventListener('resize', onChange);
     };
   }, [forceUpdate]);
+
+  useEffect(() => {
+    const aspectRatio = screen.width/screen.height;
+    setSupportedFeatures({
+      interactiveState: true,
+      aspectRatio
+    });
+  }, [initMessage]);
```

Comment:
Did the code in the CFM try to handle the case where the screen changed sized. Perhaps if the window was dragged to second screen?  I'm guessing not, but I wanted to check since it seems like this code would not handle this case.

If the old didn't handle that case then I don't think it is necessary to handle it here either.

Reply:
The old one didn't handle the change in aspect ratio when changing screen size. This one does change height/aspect ratio when screen size changes.